shared/version.go

```diff
@@ -1,5 +1,5 @@
 package shared
 
 const (
-	ConstellationVersion string = "1.0.0-b1"
+	ConstellationVersion string = "1.0.0-dev"
```

Comment:
double check that this is on purpose

Reply:
1.0.1-dev?

Reply:
Yeah this is on purpose, thanks for checking :+1: